ui/pages/send/send-content/add-recipient/add-recipient.stories.js

```diff
@@ -9,31 +8,71 @@ import AddRecipient from './add-recipient.component';
 
 const store = configureStore(testData);
 
+const { metamask } = store.getState();
+const { addressBook } = metamask;
+const recipient = metamask.accountArray[0];
+
 export default {
   title: 'Pages/Send/SendContent/AddRecipient',
   id: __filename,
   decorators: [(story) => <Provider store={store}>{story()}</Provider>],
+  argTypes: {
```

Comment:
One improvement you could make is to move the args for `ensError` and `ensWarning` here also. I feel like the default story should contain all component prop options.

Ahh so I see the dilemma here it needs to be a translation key. Maybe we could add the argType name and description options this will let the user know it should be a key. Kinda not great it shows up in the name column of the argsTable but I think it's clear not to include the brackets in the prop name...

<img width="652" alt="Screen Shot 2022-02-17 at 9 39 07 AM" src="https://user-images.githubusercontent.com/8112138/154372947-05566766-3653-4180-85eb-4dd5d09778a7.png">
<img width="1060" alt="Screen Shot 2022-02-17 at 9 39 46 AM" src="https://user-images.githubusercontent.com/8112138/154372954-d02e1a2b-894a-4a86-9b0a-6562466d5f6f.png">

```suggestion
  argTypes: {
    ensError: {
      control: {
        type: 'select',
      },
      options: ['loading'],
      name: 'ensError(must be the key for a translation)',
      description: 'must be the key for a translation',
    },
    ensWarning: {
      control: {
        type: 'select',
      },
      options: ['loading'],
      name: 'ensWarning(must be the key for a translation)',
      description: 'must be the key for a translation',
    },
```

Reply:
I like the addition of the descriptions here. We might want to keep in mind that we will likely be undoing some component prop args as class components are refactored into functional components. I am making a mental note for myself to refactor class components to functional components before writing stories when possible moving forward.